Teradata/covalent
# chore(): upgrade from ng-packager v2.0.0 to v2.4.1

## Description
Upgrade from ng-packager v2.0.0 to v2.4.1 we obtain a few important feature such as:
support intra-package dependencies (secondary intra-entry point support, doesn't look like primary intra-entry point is supported yet), allow Angular 6 as a peer dependency, and more.

### What's included?

- Upgrade from ng-packager v2.0.0 to v2.4.1

#### Test Steps (Covalent build/test)
- Verify you are using node v8.9.x
- To make the next steps a little easier go to `scripts/build-release` and add this code to the bottom of the file to `npm pack` instead of doing it manually:
```
cd deploy/platform/core
npm pack
cd ../dynamic-forms
npm pack
cd ../highlight
npm pack
cd ../http
npm pack
cd ../markdown
npm pack
```
- Verify `npm run serve` works accordingly. Test all components [http://localhost:4200/](http://localhost:4200/)
- run `npm run build:lib`
#### Test Steps (Product build)
- Clone AppCenter repo into the directory above this one.
- `yarn cache clean`
- `yarn install`
- `yarn remove @covalent/core`
- `yarn remove @covalent/highlight`
- `yarn remove @covalent/http`
- `yarn remove @covalent/markdown`
- `yarn add <file-path-to-*.tgz>` repeat for (core, highlight, http, markdown respectively) (EX: `yarn add ../covalent/deploy/platform/core/covalent-core-0.0.0-COVALENT.tgz`)
- `npm rebuild node-sass`
- Verify you are using node v8.9.x. 
- Verify `npm run serve` works correctly
- Verify `./node_modules/@angular/cli/bin/ng build --aot --prod --sourcemap=false --build-optimizer` successfully runs
- Verify `./node_modules/@angular/cli/bin/ng serve --aot --prod --sourcemap=false --build-optimizer --proxy-config proxy.conf.json` successfully runs and you can access the local site [http://localhost:4200/](http://localhost:4200/)
- Modify the import paths (primary/secondary entry point usage) in app.module and try the prod build again it should run successfully.
- Using similar steps above try it on a different product that is using covalent (WLA, ect..), make sure to remove the current node_module/@covalent/* before installing the packages from this repo.

#### General Tests for Every PR


##### Screenshots or link to StackBlitz/Plunker
